### PR TITLE
Making Docs better

### PR DESCRIPTION
**Pull request by richajaindce**
Ben Case wants to look at the docs in an easy to understand way. This PR is making cargo doc work removing errors and also making the sort protocol docs show up properly
PS: to open cargo doc with private modules use following command:
cargo doc --document-private-items --open

### CODE REVIEW

File: src/helpers/network.rs [hunk 1]
```diff
@@ -55,9 +55,9 @@ impl Debug for ChannelId {
     }
 }
 
-/// Wrapper around a [`PollSender`] to modify the error message to match what the [`Network`] trait
+/// Wrapper around a [`PollSender`] to modify the error message to match what the `Network` trait
 /// requires. The only error that [`PollSender`] will generate is "channel closed", and thus is the
-/// only error message forwarded from this [`Sink`].
+/// only error message forwarded from this `Sink`.
```
Comment (benjaminsavage) [inline src/helpers/network.rs:60]: Why did you remove the square braces from "Network" and "Sink" but not "PollSender"?

File: src/net/server/mod.rs [hunk 2]
```diff
@@ -72,14 +72,14 @@ impl MpcHelperServerError {
     }
 }
 
-/// [`From`] implementation for [`MpcServerError::InvalidHeader`]
+/// [`From`] implementation for [`MpcHelperServerError::InvalidHeader`]
```
Comment (benjaminsavage) [inline src/net/server/mod.rs:75]: Let's just remove all of these useless comments that say the same thing as the line below. They add no value and as we see here - have a tendency to fall out of sync.

File: src/net/server/mod.rs [hunk 3]
```diff
@@ -127,9 +127,9 @@ impl IntoResponse for MpcHelperServerError {
     }
 }
 
-/// Provides a mapping of [`QueryId`]s to senders that forward data to a [`Network`]. Every time a
-/// new query is started, a [`Network`] is created to handle communication for that query. When the
-/// server receives a request, it must know which [`Network`] to forward that request to, so it
+/// Provides a mapping of [`QueryId`]s to senders that forward data to a `Network`. Every time a
```
Comment (benjaminsavage) [inline src/net/server/mod.rs:130]: If we're going to be consistent about removing square braces, let's remove these as well.

File: src/net/server/mod.rs [hunk 4]
```diff
@@ -208,7 +208,7 @@ pub enum BindTarget {
 
 /// Contains all of the state needed to start the MPC server.
 /// For now, stub out gateway with simple send/receive
-/// TODO (ts): replace stub with real thing when [`Network`] is implemented
+/// TODO (ts): replace stub with real thing when `Network` is implemented
```
Comment (benjaminsavage) [inline src/net/server/mod.rs:211]: @akoshelev , @thurstonsand - can we remove this TODO now?

File: src/protocol/attribution/aggregate_credit.rs [hunk 5]
```diff
@@ -55,20 +55,20 @@ where
 /// Transpose rows of bits into bits of rows
 ///
 /// input:
-/// [
-///   [ row[0].bit0, row[0].bit1, ..., row[0].bit31 ],
-///   [ row[1].bit0, row[1].bit1, ..., row[1].bit31 ],
+/// `\[`
+///  `\[`row`\[`0`\]`.bit0,row`\[`0`\]`.bit1, ...,row`\[`0`\]`.bit31`\]`
```
Comment (benjaminsavage) [inline src/protocol/attribution/aggregate_credit.rs:59]: I *think* you can just put one backtick at the start and one at the end, and treat the entire line as a math expression.

File: src/protocol/sort/compose.rs [hunk 6]
```diff
@@ -4,24 +4,27 @@ use embed_doc_image::embed_doc_image;
...
 /// Steps
-/// ![Compose steps][compose]
+///
 /// 1. Generate random permutations using prss
 /// 2. First permutation (sigma) is shuffled with random permutations
 /// 3. Reveal the permutation
 /// 4. Revealed permutation is applied locally on another permutation shares (rho)
 /// 5. Unshuffle the permutation with the same random permutations used in step 2, to undo the effect of the shuffling
+///
+/// ![Compose steps][compose]
```
Comment (benjaminsavage) [inline src/protocol/sort/compose.rs:27]: What does this do?

File: src/helpers/network.rs (hunk 1 above)
Comment (richajaindce) [reply src/helpers/network.rs:60]: The code has changed and Network and Sink no longer exist. Tweaking the comment for now

File: src/protocol/sort/compose.rs (hunk 6 above)
Comment (richajaindce) [reply src/protocol/sort/compose.rs:27]: This shows the image in the UI like this

File: src/protocol/attribution/aggregate_credit.rs (hunk 5 above)
Comment (richajaindce) [reply src/protocol/attribution/aggregate_credit.rs:59]: Thanks for this! I wasted some time to get the backticks in :(

File: src/net/server/mod.rs (hunk 3 above)
Comment (richajaindce) [reply src/net/server/mod.rs]: I kept them around. Changed the names

**Review by akoshelev — APPROVED** [review]
